docs/api/python/getting-started.md

```diff
@@ -0,0 +1,580 @@
+from eth_account import Account
+from eth_account.signers.local import LocalAccount
+
+from zkSync2.signer.eth_signer import PrivateKeyEthSigner
+from zkSync2.transaction.transaction712 import Transaction712
+
+
+def withdraw():
+    ZKSYNC_NETWORK_URL: str = 'https://zkSync2-testnet.zkSync.dev'
+    account: LocalAccount = Account.from_key('YOUR_PRIVATE_KEY')
+    zkSync_web3 = zkSyncBuilder.build(ZKSYNC_NETWORK_URL)
+    chain_id = zkSync_web3.zkSync.chain_id
+    signer = PrivateKeyEthSigner(account, chain_id)
+    ETH_TOKEN = Token.create_eth()
+
+    nonce = zkSync_web3.zkSync.get_transaction_count(account.address, ZkBlockParams.COMMITTED.value)
+    bridges: BridgeAddresses = zkSync_web3.zkSync.zks_get_bridge_contracts()
+
+    l2_func_encoder = L2BridgeEncoder(zkSync_web3)
+    call_data = l2_func_encoder.encode_function(fn_name="withdraw", args=[
+        account.address,
+        ETH_TOKEN.l2_address,
+        ETH_TOKEN.to_int(Decimal("0.001"))
+    ])
+
+    tx = create_function_call_transaction(from_=account.address,
+                                          to=bridges.l2_eth_default_bridge,
+                                          ergs_limit=0,
+                                          ergs_price=0,
+                                          data=HexStr(call_data))
+    estimate_gas = zkSync_web3.zkSync.eth_estimate_gas(tx)
+    gas_price = zkSync_web3.zkSync.gas_price
+
+    print(f"Fee for transaction is: {estimate_gas * gas_price}")
+
+    tx_712 = Transaction712(chain_id=chain_id,
+                            nonce=nonce,
+                            gas_limit=estimate_gas,
+                            to=tx["to"],
+                            value=tx["value"],
+                            data=tx["data"],
+                            maxPriorityFeePerGas=100000000,
+                            maxFeePerGas=gas_price,
+                            from_=account.address,
+                            meta=tx["eip712Meta"])
+
+    singed_message = signer.sign_typed_data(tx_712.to_eip712_struct())
+    msg = tx_712.encode(singed_message)
+    tx_hash = zkSync_web3.zkSync.send_raw_transaction(msg)
+    tx_receipt = zkSync_web3.zkSync.wait_for_transaction_receipt(tx_hash, timeout=240, poll_latency=0.5)
+    print(f"tx status: {tx_receipt['status']}")
+
+
+if __name__ == "__main__":
+    withdraw()
+
+
+```
+
+## Deploying a contract with Create
+With zkSync, you can deploy a contract using the `create` method, by simply building the contract into a binary format and deploying it to the zkSync network.
+
+In the next steps, we will guide you through how it works.
+
+### Step1: Create a contract
+Here is a simple contract:
+
+```solidity
+
+pragma solidity ^0.8.0;
+
+contract Counter {
+    uint256 value;
+
+    function increment(uint256 x) public {
+        value += x;
+    }
+
+    function get() public view returns (uint256) {
+        return value;
+    }
+}
+
+```
+> It must be compiled by zkSync compiler only!
```

Comment:
Maybe add a link to a section in our docs where we mention how to compile contracts